### PR TITLE
Adds support for Financial Actions API

### PR DESCRIPTION
**Pull request by martinseco**

### CODE REVIEW

**Review by armandojaleo — APPROVED** [review]
Good Job!

**Review by armandojaleo — APPROVED** [review]

File: lib/checkout_sdk/checkout_api.rb [hunk 1]
```diff
@@ -35,6 +35,8 @@ module CheckoutSdk
   #   @return [CheckoutSdk::Transfers::TransfersClient]
   # @!attribute metadata
   #   @return [CheckoutSdk::Metadata::MetadataClient]
+  # @!attribute metadata
```
Comment (Armando-Ibarra) [inline lib/checkout_sdk/checkout_api.rb:38]: metadata?

File: spec/checkout_sdk/disputes/disputes_integration_spec.rb [hunk 2]
```diff
@@ -269,6 +269,5 @@
 end
 
 private def there_are_disputes(response)
-  sleep 10
```
Comment (Armando-Ibarra) [inline spec/checkout_sdk/disputes/disputes_integration_spec.rb:272]: thanks

**Review by Armando-Ibarra — APPROVED** [review]
Please just change the property hint before merge

**Pull request merged**